crates/router/src/lib.rs

```diff
@@ -175,20 +175,20 @@ fn calculate_default_headers(
     let path_info = route_match.trailing_wildcard();
 
     let scheme = req.scheme();
-    let scheme = match scheme.as_ref().unwrap_or(&Scheme::Https) {
+    let scheme = match scheme.as_ref().unwrap_or(&Scheme::Http) {
```

Comment:
I imagine this is probably fine, but it could plausibly be a breaking change.

Reply:
It's definitely a breaking change, but I don't think it actually breaks anything in practice. This is unfortunately at the edge where `spin-test` has the biggest chance to diverge from other runtimes. I imagine most Spin runtimes will always set the scheme of the request and not allow it to be `none` ever.